$NetBSD: patch-ab,v 1.1 1998/04/10 22:30:41 tron Exp $

The trailing space after the shared line breaks makesetup due to a bug
in NetBSD 1.3[.1] /bin/sh. It's fixed in current. See PR#4805 for
details.

		-- tsarna@endicor.com, 18 March 1998

--- Setup.orig	Wed Mar 18 01:19:04 1998
+++ Setup	Wed Mar 18 01:19:04 1998
@@ -1,6 +1,6 @@
 #This file is designed for building a dynamic extension.
 #See README for detailed build instructions.
-*shared* 
+*shared*
 
 # The array and ufunc object implementation
 #[VC50]cpp_options = cpp_options + ' /Tp '
